web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field--node--os2loop-question-answers.html.twig

```diff
@@ -37,7 +38,7 @@
     <h2{{ content_attributes }}>{{ 'Add new comment'|t }}</h2>
     <div class="user">
       {% include '@os2loop_theme/field/user-image.html.twig' with {'uid': user.account.id} %}
-      {% include '@os2loop_theme/field/username.html.twig' with {'uid': user.account.id, 'account': logged_in_user} %}
+      {% include '@os2loop_theme/field/user-name.html.twig' with {'uid': user.account.id} %}
     </div>
```

Comment:
These 4 lines are duplicated in (at least) two template files and could be refactored into a shared template, `user-info.html.twig`, say. Furthermore, we should pass the entire `user` object to the template and let it get the parts it needs, e.g.

```
{% include '@os2loop_theme/user-info.html.twig' with {user: user} %}
```

Reply:
A user-info.html.twig has been added.